test/gtest/ucs/test_memtrack.cc

```diff
@@ -159,12 +159,15 @@ UCS_TEST_F(test_memtrack, sysv) {
 
 UCS_TEST_F(test_memtrack, memalign_realloc) {
     void* ptr;
+    int ret;
 
-    ptr = ucs_memalign(10, ALLOC_SIZE, ALLOC_NAME);
+    ret = ucs_posix_memalign(&ptr, 8, ALLOC_SIZE, ALLOC_NAME);
+    ASSERT_EQ(0, ret);
     ASSERT_NE((void *)NULL, ptr);
     ucs_free(ptr);
 
-    ptr = ucs_memalign(1000, ALLOC_SIZE, ALLOC_NAME);
+    ret = ucs_posix_memalign(&ptr, 1024, ALLOC_SIZE, ALLOC_NAME);
```

Comment:
probably need to add coverity suppression here, because it doesn't understand ptr was reallocated by ucs_posix_memalign
or maybe just set ptr to NULL after the ucs_free in line 167

Reply:
Can you point me to the coverity diagnostic ?  Or put it inline there, please.

Reply:
http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/label=hpc-test-node-legacy,worker=0/11927//artifact/cov_build_devel_11927/output/errors/index.html